android/src/main/java/com/rudderstack/android/internal/infrastructure/ActivityBroadcasterPlugin.kt

```diff
@@ -65,6 +64,7 @@ internal class ActivityBroadcasterPlugin(
                         broadCastApplicationStop()
                     }
                 }
+                analytics?.flush()
```

Comment:
move to LifecycleObserverPlugin
add a test case.

Reply:
I've addressed your comments. Please review it again.